### DISCUSSION TITLE
Getting warning with getText()

### DISCUSSION THREAD

**Discussion by vijuc895** [pymupdf/PyMuPDF#1267] [Q&A] [answered]
Hello @JorjMcKie,

I am getting warning with getText() for one of my pdf, the warning is `mupdf: zlib error: incorrect header check. ` and instead of text I am getting `'��� ����� ���� ���\n` whereas when I open this pdf in any browser and try to select the text and copy it then I am getting proper text.
<img width="532" alt="Screenshot 2021-09-15 at 11 43 37 PM" src="https://user-images.githubusercontent.com/43551239/133487160-298eb9ec-90f9-4c35-9212-3d58a6329dd7.png">

**Comment by JorjMcKie** [comment]
Re-submit with reproducible data and all the rest of mandatory information

**Comment by JorjMcKie** [comment, marked as answer]
Probably duplicate of #869 anyway.
The zlib warnings indicate document damage.

Please also see [here](https://pymupdf.readthedocs.io/en/latest/faq.html#missing-or-unreadable-extracted-text).
